rotkehlchen/tests/utils/exchanges.py

```diff
@@ -291,6 +292,27 @@ def create_test_binance(
     return binance
 
 
+def create_test_binance_us(
```

Comment:
This is not used anywhere, right? So just remove it. No need for this create function.

Reply:
I have to double check as some API exchanges  tests look up for the `create_test_<exchange_name>` and there wasn't one for Binance US. Besides it wasn't clear yesterday if I had to implement specific tests for Binance US. I'll delete it.